nupic/research/frameworks/vernon/parser_utils.py

```diff
@@ -151,6 +145,11 @@ def process_args(args, config):
 
     if "wandb" in args and args.wandb:
 
+        from wandb import util
+        from nupic.research.frameworks.wandb import ray_wandb
+        from nupic.research.frameworks.dynamic_sparse.common.ray_custom_loggers_2 \
```

Comment:
What is the relationship between `nupic.research.frameworks.dynamic_sparse` and `wandb`?

Reply:
That is a good point. The reason I moved that there is that the custom loggers are only used inside the `if wandb`. We should probably import those from someplace else other than dynamic sparse, I can put that in a separate PR. This one just moves the imports where they are used to reduce dependencies.

Reply:
Luiz I ended up fixing it here, it is a very small change.